### PR TITLE
Unify headings titles and breadcrumbs for Core docs 2.1.5 - closes #477

### LINKED ISSUE #477: Unify headings, titles and breadcrumbs for Core 2.1.5
- Same page title as the navigation title of the page for every page 
- Update navigations to not use bold navigation links 
- reference/configuration reference: change title to be only "configuration"
- add banner images for Core (like in the README files on GitHub)
- Use imperative form in page titles, e.g.: Config file handling --> Handle config files
- Use sentence case in titles (small letters except first letter of first word, names, abbreviations,...)
- Change titles to remove redundancy: 
  - `Setup/Binary installation` to `Setup/Binary`
  - `Setup/Docker installation` to `Setup/Docker`
  ...
  - `Update/Binary update` to `Update/Binary`
  ... and so on

### PR DESCRIPTION
**Pull request by chris529**
closes #477

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 * xref:interact-with-network.adoc[Interact with the network]
 * *Setup*
-** xref:setup/binary.adoc[Binary installation]
-** xref:setup/commander.adoc[Commander installation]
-** xref:setup/docker.adoc[Docker installation]
-** xref:setup/source.adoc[Source installation]
+** xref:setup/binary.adoc[Setup Binary]
+** xref:setup/commander.adoc[Setup Commander]
+** xref:setup/docker.adoc[Setup Docker]
+** xref:setup/source.adoc[Setup Source]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: Remove redundancy in titles: 

```suggestion
** xref:setup/binary.adoc[Binary]
** xref:setup/commander.adoc[Commander]
** xref:setup/docker.adoc[Docker]
** xref:setup/source.adoc[Source]
```

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -16,10 +16,10 @@
...
 ** xref:management/ssl.adoc[Enable SSL]
 * *Update*
-** xref:update/binary.adoc[Binary update]
-** xref:update/commander.adoc[Commander update]
-** xref:update/docker.adoc[Docker update]
-** xref:update/source.adoc[Source update]
+** xref:update/binary.adoc[Update Binary]
+** xref:update/commander.adoc[Update Commander]
+** xref:update/docker.adoc[Update Docker]
+** xref:update/source.adoc[Update Source]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]:
```suggestion
** xref:update/binary.adoc[Binary]
** xref:update/commander.adoc[Commander]
** xref:update/docker.adoc[Docker]
** xref:update/source.adoc[Source]
```

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 * *Setup*
-** xref:setup/binary.adoc[Binary installation]
-** xref:setup/commander.adoc[Commander installation]
-** xref:setup/docker.adoc[Docker installation]
-** xref:setup/source.adoc[Source installation]
+** xref:setup/binary.adoc[Setup Binary]
+** xref:setup/commander.adoc[Setup Commander]
+** xref:setup/docker.adoc[Setup Docker]
+** xref:setup/source.adoc[Setup Source]
 * *Management*
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: Please remove boldness from all navigation links
```suggestion
* Management
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -34,21 +35,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_getting_started: getting-started.adoc
 :url_config_api: management/api-access.adoc
 
+:image:lisk_docu_blog_core.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: Use this banner: https://github.com/LiskHQ/lisk-core

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -243,14 +246,14 @@ For example, the _protocol version_ is used in P2P communication between Lisk Co
 [[upgrade_vs_migration]]
 == Update vs Migration
 
-When to update, and when to migrate Lisk Core?
+When to update, and when to migrate to Lisk Core?
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
When to update, and when to migrate Lisk Core?
```

We are migrating Lisk Core to a new version, it should be correct like this.

File: modules/ROOT/pages/interact-with-network.adoc [hunk 6]
```diff
@@ -53,24 +53,24 @@ However, in this case it may be necesssary to customize the tools in order to fu
 It is possible to choose up to 5 different methods to interact with an existing network, based on the most convenient and appropriate requirements as described below:
 
 [[javascript]]
-=== A - Write scripts in Javascript
+=== A - Write scripts in javascript
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-network.adoc]:
```suggestion
=== A - Write scripts in JavaScript
```

Please use this capitalisation everywhere

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -34,21 +35,23 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+
 ifeval::[{page-component-version} !== master]
 IMPORTANT: To access the latest Lisk Core version which has been updated, please xref:master@{page-component-name}::{page-relative}[Click here].
 endif::[]
 
-== Introduction
 
-The Lisk Core is the program that actually implements the Lisk Protocol.
+
+The Lisk Core is the program that actually implements the Lisk protocol.
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Lisk Core is the program that actually implements the Lisk protocol.
```

File: modules/ROOT/pages/setup/binary.adoc [hunk 8]
```diff
@@ -1,4 +1,4 @@
-= Lisk Core Binary Setup
+= Setup Binary
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc]:
```suggestion
= Binary
```

Please remove redundant titles on the other pages as well.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: done

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -34,21 +35,23 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_getting_started: getting-started.adoc
 :url_config_api: management/api-access.adoc
 
+:image:banner_core.png[Logo]
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: The image is missing.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 9 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: OK it should be there now.. ;-)

File: modules/ROOT/nav.adoc [hunk 10]
```diff
@@ -1,11 +1,11 @@
 * xref:getting-started.adoc[Getting started]
 * xref:interact-with-network.adoc[Interact with the network]
 * *Setup*
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: Please remove bold styling here and on all other bold navigation items

File: modules/ROOT/pages/index.adoc (hunk 9 above)
Comment (Tschakki) [reply modules/ROOT/pages/index.adoc]: Ok nice! Only one small issue:
```suggestion
image:banner_core.png[Logo]
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Nearly good!  Just 2 small issues, see comments.

File: modules/ROOT/nav.adoc (hunk 10 above)
Comment (chris529) [reply modules/ROOT/nav.adoc]: ok it is done..

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**